Formula/yorkie.rb

```diff
@@ -0,0 +1,36 @@
+class Yorkie < Formula
+  desc "Document store for collaborative applications"
+  homepage "https://yorkie.dev/"
+  url "https://github.com/yorkie-team/yorkie.git",
+    tag:      "v0.2.6",
+    revision: "284ceddfaa6400ce6eaaffad4570393c729ebf81"
+  license "Apache-2.0"
+  head "https://github.com/yorkie-team/yorkie.git", branch: "main"
+
+  depends_on "go" => :build
+
+  def install
+    system "make", "build"
+    bin.install Dir[buildpath/"bin/*"]
+  end
+
+  service do
+    run opt_bin/"yorkie"
+    run_type :immediate
+    keep_alive true
+    working_dir var
+  end
+
+  test do
+    yorkie_pid = fork do
+      exec bin/"yorkie", "server"
+    end
+    # sleep to let yorkie get ready
+    sleep 3
+
+    shell_output("#{bin}/yorkie project ls")
```

Comment:
Can we check some output here?

Reply:
@SMillerDev Thanks for the good comments. I added output check logic.